### DISCUSSION TITLE
Full compatibility with Windows 11 and other issues

### DISCUSSION THREAD

**Discussion by matheus-j-ressurreicao** [GrandOrgue/grandorgue#911] [Developers] [answered]
Hello, first of all I would like to thank you very much for the revitalization of this wonderful software that is GrandOrgue. Congratulations on the work of all the developers who have brought this project back to life. I am very much looking forward to the features that will come in the next releases and I hope that GrandOrgue will soon become a major reference when it comes to the virtual organ. In this spirit, I would very much like to ask, in view of the most recent hardware and software updates, about full compatibility with Windows 11 (issues involving application compatibility, audio engine, etc.)? Are there any plans to update the core software in view of the latest hardware and software updates? I apologize if this is not the appropriate channel (this is my first time commenting on Github). Again, I register here a big thanks to the whole development team and hope that life will be revitalized for GrandOrgue. Thank you very much.

**Comment by oleg68** [comment, marked as answer]
> about full compatibility with Windows 11 (issues involving application compatibility, audio engine, etc.)? Are there any plans to update the core software in view of the latest hardware and software updates?

We hope that the existing GrandOrgue application is fully compatible with Windows 11, so we do not have any plans to update something in this order.

But feel free to create issues at github if something does not work on Windows 11 as expected.

**Comment by matheus-j-ressurreicao** [comment]
Oleg, thank you very much for the clarification. As I mentioned, I am very happy with the revitalization that you and the new development team have been bringing to GrandOrgue! Please keep up this wonderful work!!! In that spirit, and even speaking of Windows 11, I wonder if I could help in some way by posting some more technical document around here in the appropriate place. For example, sometimes some software requests submission of memory dump data or other data for development purposes. In that spirit, if I can help in any way by submitting this data and if it will contribute in any way to the next GrandOrgue development work, please tell me the appropriate channel and what to do and I will do it. Oh, and if there are any problems in W11, I will let you know in the appropriate channel. Oleg, I reinforce my thanks, and renew my protestations of esteem and consideration.

**Comment by oleg68** [comment]
@matheus-j-ressurreicao thank you. The special pace for bug reports and crash dump reports is here: https://github.com/GrandOrgue/grandorgue/issues

If you can test GrandOrgue under Windows 11, it would be great.
